Source/ee/Vpu.h

```diff
@@ -39,7 +40,17 @@ class CVpu
 							CVpu(unsigned int, const VPUINIT&, CGIF&, uint8*, uint8*);
 	virtual					~CVpu();
 
-	void					Execute(bool);
+	void* operator new(size_t allocSize)
+	{
+		return framework_aligned_alloc(allocSize, 0x10);
```

Comment:
I added it because without it visual studio warns that the object on the heap will not be 16 byte aligned. There was no reason that I would see - I searched for __aligned and the relevant declspec but could not see any. There must be some object in the class structure though that is specify this alignment though or the compiler would not give the warning.